src/domains/wallet/components/WalletsGroup/WalletsGroup.blocks.tsx

```diff
@@ -169,6 +160,11 @@ export const GroupNetworkTotal: React.VFC<WalletsGroupNetworkTotalProperties> =
 			return <GroupSkeleton width={110} className="h-4" innerClassName="h-3.5" />;
 		}
 
+		const totalConvertedNetworkBalance = wallets.reduce(
+			(balance, wallet) => balance + wallet.convertedBalance(),
+			0,
+		);
+
 		return <Amount value={totalConvertedNetworkBalance} ticker={exchangeCurrency} />;
```

Comment:
@alfonsobries do you think we could use BigNumber to avoid any floating point arithmetic overflow :hmhmhm: 

```suggestion
		const totalNetworkBalance = wallets.reduce((balance, wallet) => balance.plus(wallet.balance()), BigNumber.ZERO);

		return <Amount value={totalNetworkBalance.toNumber()} ticker={network.ticker()} />;

```